props/saber_shtok_buttons.h

```diff
@@ -1,47 +1,98 @@
 // Play/Stop Music - hold 1 second and release Activation button while OFF or ON
-// Turn the blade off - hold and wait till blade is off while ON (like in Plecter boards)
+// Turn the blade off - hold and wait till blade is off while ON (like in Plecter boards) or twist the saber like a bike handle
 // Next Preset - short click AUX button while OFF
 // Previous Preset - hold AUX and click Activation button while OFF
 // Lockup - hold AUX button while ON (like in Plecter boards)
 // Drag - hold AUX button while ON pointing the blade tip down
 // Blaster Blocks - short click AUX button while ON
-// Force Effects - short click Activation button while ON
-// Enter Color Change mode - hold AUX and click Activation button while ON
-// Confirm selected color in Color Change mode - hold the button until confirmation sound
+// Force Effects - short click Activation button while ON or perform "push" gesture holding the hilt vertically
+// Enter Color Change mode - hold AUX and quickly press and release Activation button while ON then release AUX button
+// Confirm selected color and exit Color Change mode - hold the button until confirmation sound
 // Melt - hold AUX (or Activation) button + perform stab action while ON
 // Lightning Block - hold Activation button + short click AUX button while ON
-// Enter Multi-Block mode - swing + hold the button for 1 second and release while ON (now swing the saber, blaster blocks will trigger automatically)
+// Enter Multi-Block mode - hold the Activation button + swing the saber and release the button while ON (now swing the saber, blaster blocks will trigger automatically)
 // Exit Multi-Block mode - short click AUX button while ON
+// Enter Battle Mode - hold the AUX button + swing the saber and release the button while ON or hold the Activation button + use "Gesture/Swing Ignition" then release the button while OFF (blade will turn ON in Battle Mode)
+// Exit Battle Mode - hold the AUX button + swing the saber and release the button while ON or turn the blade OFF
+//
+//
+// CUSTOM SOUNDS SUPPORTED (add to font to enable):
+//
+// On Demand Power Save - dim.wav
+// On Demand Battery Level - battery.wav
+// Battle Mode On (on toggle) - bmbegin.wav
+// Battle Mode Off (on toggle) - bmend.wav
+// Enter Volume Menu - vmbegin.wav
+// Exit Volume Menu - vmend.wav
+// Force Push - push.wav
+// Fast On (optional) - faston.wav
+// Multi-Blast Mode On - blstbgn.wav
+// Multi-Blast Mode Off - blstend.wav
 
 #ifndef PROPS_SABER_SHTOK_BUTTONS_H
 #define PROPS_SABER_SHTOK_BUTTONS_H
 
+#ifndef MOTION_TIMEOUT
+#define MOTION_TIMEOUT 60 * 5 * 1000
+#endif
+
 #include "prop_base.h"
+#include "../sound/hybrid_font.h"
 
 #undef PROP_TYPE
 #define PROP_TYPE SaberShtokButtons
 
+EFFECT(dim); // for EFFECT_POWERSAVE
+EFFECT(battery); // for EFFECT_BATTERY_LEVEL
+EFFECT(bmbegin); // for Begin Battle Mode
+EFFECT(bmend); // for End Battle Mode
+EFFECT(vmbegin); // for Begin Volume Menu
```

Comment:
SFX_vmbegin and SFX_vmend doesn't seem to be used anywhere.

Reply:
Deleted.

Reply:
Looks like they are still here to me.

Reply:
Now should be deleted.